### ISSUE TITLE
unexpected problem in running autoformer

### ISSUE THREAD

**Issue by SpannrLOL** [thuml/Autoformer#85]
![1](https://user-images.githubusercontent.com/114562151/192707618-c003fd32-f5ce-43ca-b84d-5700f6df0c7c.png)
I just use the data electricity. xsl to run the autoformer model, then there's an unexpected problem in it. Could you please inform me why?
The basic input are as follows:
(activation='gelu', batch_size=32, bucket_size=4, c_out=7, checkpoints='./checkpoints/', d_ff=2048, d_layers=1, d_model=512, data='ETTh2', data_path='electricity.csv', dec_in=7, des='test', devices='0,1,2,3', distil=True, do_predict=False, dropout=0.05, e_layers=2, embed='timeF', enc_in=7, factor=1, features='M', freq='h', gpu=0, is_training=1, itr=2, label_len=48, learning_rate=0.0001, loss='mse', lradj='type1', model='Autoformer', model_id='test', moving_avg=25, n_hashes=4, n_heads=8, num_workers=10, output_attention=False, patience=3, pred_len=96, root_path='./dataset/electricity/', seq_len=96, target='OT', train_epochs=10, use_amp=False, use_gpu=True, use_multi_gpu=False)
Use GPU: cuda:0
